invenio_rdm_records/services/schemas/metadata.py

```diff
@@ -72,50 +65,26 @@ class CreatibutorSchema(Schema):
     name = SanitizedUnicode()
     given_name = SanitizedUnicode()
     family_name = SanitizedUnicode()
-    identifiers = fields.Dict()
+    identifiers = IdentifierSet(
+        fields.Nested(partial(
+            IdentifierSchema,
+            allowed_schemes=[*ORGANIZATIONAL_SCHEMES, *PERSONAL_SCHEMES]
+        ))
+    )
     affiliations = fields.List(fields.Nested(AffiliationSchema))
 
-    @validates("identifiers")
-    def validate_identifiers(self, value):
-        """Validate well-formed identifiers are passed."""
-        schemes = ['orcid', 'ror']
-
-        if any(scheme not in schemes for scheme in value.keys()):
-            raise ValidationError(
-                [_(f"Invalid value. Choose one of {schemes}.")]
-            )
-
-        if 'orcid' in value:
-            if not idutils.is_orcid(value.get('orcid')):
-                raise ValidationError({'orcid': [_("Invalid value.")]})
-
-        if 'ror' in value:
-            if not idutils.is_ror(value.get('ror')):
-                raise ValidationError({'ror': [_("Invalid value.")]})
-
     @validates_schema
-    def validate_type_identifiers(self, data, **kwargs):
-        """Validate identifier based on type."""
-        if data['type'] == "personal":
-            person_identifiers = ['orcid']
-            identifiers = data.get('identifiers', {}).keys()
-            if any([i not in person_identifiers for i in identifiers]):
-                messages = [
-                    _(f"Invalid value. Choose one of {person_identifiers}.")
-                ]
-                raise ValidationError({"identifiers": messages})
+    def validate_identifiers_scheme(self, data, **kwargs):
```

Comment:
Minor: First of all, it might just be me being confused here 🤔 What's the purpose of this method and why is it not already validated by ``IdentifierSet`` and ``IdentifierSchema``?

Reply:
This is the sort of "extra" case we talked IRL about `ror` only for organizations and `isni/orcid` only for person. The thing is at `IdentifierSchema` level it checks an allowed list of schemes... but knowing the type and this extra check is responsability of the `PersonOrOrgSchema`...

Reply:
Discussed IRL, opted to remove this validation. The reasons are:

- Migrating previous data (e.g. Zenodo, and Zenodo-like clones) that for one reason or another did not check this and will have a very difficult time importing data.
- From the UI we can control this by not displaying the PID schemes, and in the backend the correct one will be selected.
- Downside: from the API they can be inserted wrongly... a leap of faith here.